docker/entrypoint.sh

```diff
@@ -12,9 +12,16 @@ trap catchall INT EXIT TERM
 
 # Stage 1, as root
 if [ $EUID = 0 ] ; then
-	# Align local temboard user with remote postgres user.
+	# Fix UID and GID
+	groupmod -g $(stat -c "%g" /var/run/docker.sock) docker
 	usermod -u $(stat -c "%u" /var/lib/postgresql/data) temboard
-	groupmod -g $(stat -c "%g" /var/lib/postgresql/data) temboard
+	if ! group=$(stat -c "%G" /var/lib/postgresql/data) ; then
+		groupmod -g $(stat -c "%g" /var/lib/postgresql/data) temboard
+		group=temboard
+	fi
+	if ! groups temboard | grep -q $group ; then
+		adduser temboard $group
+	fi
```

Comment:
Do we need to put this in a conditional statement? IMO it doesn't fail if `temboard` is already a member of the group `temboard`.